app/views/members/_member_page_header.html.haml

```diff
@@ -14,11 +14,13 @@
             - if member.person.rebellions_fraction
               %span.member-rebellions.object-data-rebellion
                 -if member.person.rebellions_fraction == 0
-                  Never rebels
+                  = "Never rebels" if member.currently_in_parliament?
+                  = "Never rebelled" if !member.currently_in_parliament?
```

Comment:
A simpler way to implement this would be to use the [ternary operator](https://en.wikipedia.org/wiki/%3F:#Ruby) in ruby (e.g. a one line if/else statement) 

```
- member.currently_in_parliament? ? "Never rebels" : "Never rebelled"
```

That would also be a more expressive way to capture this logic.